airflow/api/client/json_client.py

```diff
@@ -31,12 +31,12 @@ def _request(self, url, method='GET', json=None):
         }
         if json is not None:
             params['json'] = json
-        resp = getattr(self._session, method.lower())(**params)  # pylint: disable=not-callable
+        resp = getattr(self._session, method.lower())(**params)
         if not resp.ok:
             # It is justified here because there might be many resp types.
             try:
                 data = resp.json()
-            except Exception:  # noqa pylint: disable=broad-except
+            except Exception:  # noqa
```

Comment:
I suspect that this `noqa` tag has no effect on any tool now.  We should use [`yesqa`](https://github.com/asottile/yesqa) to delete the unused tag, but we can do it as a separate PR.

Reply:
I think flake also complains about this case. But yeah. yesqa is a good idea. I will run it now no problem.

Reply:
Flake dont support this case. In other place, we have broad exception without `noqa` tag.